runtime/src/lib.rs

```diff
@@ -116,7 +116,7 @@ construct_runtime!(
 		// Council: pallet_collective::<Instance1> = 13,
 		TechnicalCommittee: pallet_collective::<Instance2> = 14,
 		// Elections: pallet_elections_phragmen = 15,
-		TechnicalMembership: pallet_membership::<Instance1> = 16,
+		// TechnicalMembership: pallet_membership::<Instance1> = 16,
```

Comment:
What was this?

Reply:
It was to manage the technical council members, but we realised its not of great use to us